src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Permission/Blocks/profile_tab_contents.html.twig

```diff
@@ -0,0 +1,127 @@
+              </div>
+            </div>
+          </div>
+        {% else %}
+          {% set profileTypes = {
+            'view': {
+              'value': configurablePermissions.isBulkViewConfigurationEnabled(profile.id),
+              'label': ('View'|trans({}, 'Admin.Actions')),
+            },
+            'add': {
+              'value': configurablePermissions.isBulkAddConfigurationEnabled(profile.id),
+              'label': ('Add'|trans({}, 'Admin.Actions')),
+            },
+            'edit': {
+              'value': configurablePermissions.isBulkEditConfigurationEnabled(profile.id),
+              'label': ('Edit'|trans({}, 'Admin.Actions')),
+            },
+            'delete': {
+              'value': configurablePermissions.isBulkDeleteConfigurationEnabled(profile.id),
+              'label': ('Delete'|trans({}, 'Admin.Actions')),
+            },
+            'all': {
+              'value': configurablePermissions.isBulkAllConfigurationEnabled(profile.id),
+              'label': ('All'|trans({}, 'Admin.Actions')),
+            }
+            }
+          %}
+          <div class="col">
+            <div
+              id="profile-content-{{ profile.id }}"
+              data-update-url="{{ path('admin_permissions_update_tab_permissions') }}"
+              data-can-edit="{{ configurablePermissions.hasEmployeeEditPermission ? 'true' : 'false' }}"
+              data-types="{{ profileTypes|json_encode }}"
+              data-permissions="{{ configurablePermissions.tabs|json_encode}}"
+              data-title="{{ 'Menu'|trans({}, 'Admin.Advparameters.Feature')|escape }}"
+              data-empty="{{ 'No menu'|trans({}, 'Admin.Advparameters.Feature')|escape }}"
+            >
+            </div>
+          </div>
+
+          {% set moduleTypes = {
+            'view': {
+              'value': configurablePermissions.isBulkViewConfigurationEnabled(profile.id),
+              'label': ('View'|trans({}, 'Admin.Actions')),
+            },
+            'configure': {
+              'value': configurablePermissions.isBulkAddConfigurationEnabled(profile.id),
+              'label': ('Configure'|trans({}, 'Admin.Actions')),
+            },
+            'uninstall': {
+              'value': configurablePermissions.isBulkEditConfigurationEnabled(profile.id),
+              'label': ('Uninstall'|trans({}, 'Admin.Actions')),
+            },
+            }
+          %}
+          <div class="col">
+            <div
+              id="module-content-{{ profile.id }}"
+              data-update-url="{{ path('admin_permissions_update_module_permissions') }}"
+              data-can-edit="{{ configurablePermissions.hasEmployeeEditPermission ? 'true' : 'false' }}"
+              data-types="{{ moduleTypes|json_encode }}"
+              data-permissions="{{ profilePermissionsForModules[profile.id]|json_encode}}"
+              data-title="{{ 'Modules'|trans({}, 'Admin.Advparameters.Feature')|escape }}"
```

Comment:
Please localize it in _Admin.Global_, thanks.

Reply:
done